examples/analytics.amp.html

```diff
@@ -127,6 +127,19 @@
 </script>
 </amp-analytics>
 
+<!-- Quantcast tracking -->
+<amp-analytics type="quantcast">
+<script type="application/json">
+{
+  "vars": {
+    "pcode": "p-1234567890abc",
+    "labels": ["Example.label.1", "Example.label.2"]
```

Comment:
@rudygalfi  This works as it is today.  It just gets treated as a single string.  However I do have an issue open here  #1886 which would add a small change in functionality for arrays.